### PR TITLE
Fix and extend LAMP tests for PHP5 and PHP7

### PR DESCRIPTION
**Pull request by mimi1vx**
Extend LAMP to test PHP7 , and make tests independent on test order

SLE: http://argus.suse.cz/tests/1024
openSUSE: http://argus.suse.cz/tests/1025

### CODE REVIEW

File: tests/console/php5.pm [hunk 1]
```diff
@@ -1,36 +1,26 @@
 # SUSE's openQA tests
 #
-# Copyright © 2016 SUSE LLC
+# Copyright © 2017 SUSE LLC
```
Comment (dzedro) [inline tests/console/php5.pm]: 2016-2017

**Comment by okurz** [comment]
Looks very nice and clean. It would be awesome if you can also crosscheck openSUSE tests with that.

What about the failing modules in your validation run?

**Comment by mimi1vx** [comment]
@okurz openSUSE probadly will works without any problems:) Failing is due existing regression in MariaDB, fix is on way in https://build.suse.de/request/show/132926

**Comment by okurz** [comment]
LGTM. Please add test for openSUSE as well, you know Factory first :-)

File: lib/apachetest.pm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # SUSE's Apache tests
 #
-# Copyright © 2016 SUSE LLC
+# Copyright © 2017 SUSE LLC
```
Comment (asmorodskyi) [inline lib/apachetest.pm]: Normally we not replace year but extend it , 2016-2017

File: lib/apachetest.pm [hunk 3]
```diff
@@ -100,13 +120,90 @@ sub setup_apache2 {
...
+    }
+}
+sub setup_pgsqldb {
+    # without changing current working directory we get:
+    # 'could not change directory to "/root": Permission denied'
+    assert_script_run 'pushd /tmp';
+
+    # requires running postgresql94 server
+    # test basic functionality - require postgresql94
+    assert_script_run "sudo -u postgres createdb openQAdb";
```
Comment (asmorodskyi) [inline lib/apachetest.pm]: What if we will extract this to *.sql file which we can put into data directory ? I mean all CREATE/INSERT/SELECT lines too ?

**Comment by mimi1vx** [comment]
@okurz openSUSE part added , @asmorodskyi  -- createdb part moved to sql file

**Comment by okurz** [comment]
awesome work! Thank you

**Pull request merged**